### ISSUE TITLE
Doesn't work in Big Sur (((

### ISSUE THREAD

**Issue by ArturKontrabaev** [dado3212/spaces-renamer#74]
Downloaded latest version, followed all instructions and it doesn't change names of spaces (

**Comment by lotu** [comment]
I found it did not work in Big Sur v 11.1 on my Retina 27in iMac 2019.   Space renames appears to install correctlly and the icon appears in the upper right corner.  I can click on and enter new space names however, the update names button is grayed out and nothing happens when it is pressed.

**Comment by bomtailey** [comment]
I've never gotten this to work with Big Sur. I also am running Big Sur v 11.1 and spaces-renamer still doesn't change space names.

**Comment by dado3212** [comment]
The Update Names button being greyed out is a good flag that I should fix, but it's working for me on v11.1, 2019 MacBook Pro.  Can you paste the output of running `cat ~/Library/Containers/com.alexbeals.spacesrenamer/com.alexbeals.spacesrenamer.plist` and `cat ~/Library/Containers/com.alexbeals.spacesrenamer/com.alexbeals.spacesrenamer.currentspaces.plist` in Terminal?

**Comment by lotu** [comment]
$ cat ~/Library/Containers/com.alexbeals.spacesrenamer/com.alexbeals.spacesrenamer.plist
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>spaces_renaming</key>
	<dict>
		<key>14D0CFFA-D86B-4097-B638-B8EC48CE573B</key>
		<string></string>
		<key>1DE0C2ED-6867-4265-885F-190B1C4925F9</key>
		<string></string>
		<key>339B9F21-49E3-4D52-98BE-592A3D50483F</key>
		<string>Desktop 4</string>
		<key>36203216-176E-4064-B676-CAEBD2C387A0</key>
		<string>Email</string>
		<key>552AB003-F2A0-4BFA-971C-4395459768E2</key>
		<string>Desktop 6</string>
		<key>67D9D821-73AE-40FE-A3CF-B59C67B60D86</key>
		<string>Skype/Zoom</string>
		<key>7AB4F9C8-16F7-428D-9486-4BE9421372FE</key>
		<string>Desktop 8</string>
		<key>924F808D-E93A-4767-A597-B65916155DD1</key>
		<string>Desktop 7</string>
		<key>B6469B88-AA02-4D87-A09A-B973E18DAA1D</key>
		<string>Desktop 5</string>
		<key>CD10B406-1036-440C-9C9B-03E3EFF3FE35</key>
		<string>Discord</string>
		<key>DA0A10BA-6D08-4855-9F22-15D9E390A90B</key>
		<string>OBS</string>
		<key>EAF17BDC-F907-4B4A-B2B7-45712D557DCB</key>
		<string>Desktop 10</string>
		<key>FFB0B975-9BB6-4DC8-B368-C071A35FA85B</key>
		<string>Desktop 9</string>
	</dict>
</dict>
</plist>
$ cat ~/Library/Containers/com.alexbeals.spacesrenamer/com.alexbeals.spacesrenamer.currentspaces.plist
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Monitors</key>
	<array>
		<dict>
			<key>Current Space</key>
			<dict>
				<key>ManagedSpaceID</key>
				<integer>13</integer>
				<key>id64</key>
				<integer>13</integer>
				<key>type</key>
				<integer>0</integer>
				<key>uuid</key>
				<string>36203216-176E-4064-B676-CAEBD2C387A0</string>
			</dict>
			<key>Display Identifier</key>
			<string>223EDABC-93F1-01FA-D593-B169C64F160A</string>
			<key>Spaces</key>
			<array>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>3</integer>
					<key>id64</key>
					<integer>3</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>67D9D821-73AE-40FE-A3CF-B59C67B60D86</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>214</integer>
					<key>TileLayoutManager</key>
					<dict>
						<key>Age</key>
						<real>151770.24278695899</real>
						<key>ConfigAge</key>
						<real>151770.240209713</real>
						<key>Inter-Tile Spacing</key>
						<dict>
							<key>Height</key>
							<real>12</real>
							<key>Width</key>
							<real>12</real>
						</dict>
						<key>Layout Rect</key>
						<dict>
							<key>Height</key>
							<real>1620</real>
							<key>Width</key>
							<real>2880</real>
							<key>X</key>
							<real>0.0</real>
							<key>Y</key>
							<real>0.0</real>
						</dict>
						<key>Max Tile Config</key>
						<dict>
							<key>Height</key>
							<real>1</real>
							<key>Width</key>
							<real>2</real>
						</dict>
						<key>PreferredLayout</key>
						<array>
							<dict>
								<key>DisplaySize</key>
								<dict>
									<key>Height</key>
									<real>1620</real>
									<key>Width</key>
									<real>2880</real>
								</dict>
								<key>FirstTileSize</key>
								<dict>
									<key>Height</key>
									<real>1620</real>
									<key>Width</key>
									<real>2880</real>
								</dict>
							</dict>
						</array>
						<key>ReservedArea</key>
						<dict>
							<key>Bottom</key>
							<real>0.0</real>
							<key>Left</key>
							<real>0.0</real>
							<key>Right</key>
							<real>0.0</real>
							<key>Top</key>
							<real>0.0</real>
						</dict>
						<key>TileSpaces</key>
						<array>
							<dict>
								<key>ManagedSpaceID</key>
								<integer>214</integer>
								<key>SizeConstraints</key>
								<dict>
									<key>Max</key>
									<dict>
										<key>Height</key>
										<real>100000</real>
										<key>Width</key>
										<real>100000</real>
									</dict>
									<key>Min</key>
									<dict>
										<key>Height</key>
										<real>472</real>
										<key>Width</key>
										<real>940</real>
									</dict>
									<key>Preferred</key>
									<dict>
										<key>Height</key>
										<real>1506</real>
										<key>Width</key>
										<real>1592</real>
									</dict>
								</dict>
								<key>TileLimitedClipping</key>
								<true/>
								<key>TileRect</key>
								<dict>
									<key>Height</key>
									<real>1620</real>
									<key>Width</key>
									<real>2880</real>
									<key>X</key>
									<real>0.0</real>
									<key>Y</key>
									<real>0.0</real>
								</dict>
								<key>TileType</key>
								<string>Primary</string>
								<key>TileWindowID</key>
								<integer>3534</integer>
								<key>appName</key>
								<string>Discord</string>
								<key>com.apple.appkit.disablemcexit</key>
								<false/>
								<key>fromSpace</key>
								<string>339B9F21-49E3-4D52-98BE-592A3D50483F</string>
								<key>fs_wid</key>
								<integer>3534</integer>
								<key>id64</key>
								<integer>216</integer>
								<key>name</key>
								<string>#deadfalls - Discord</string>
								<key>pid</key>
								<integer>379</integer>
								<key>type</key>
								<integer>5</integer>
								<key>uuid</key>
								<string>5A24AE54-BED6-4A98-8538-69103847E8FE</string>
							</dict>
						</array>
					</dict>
					<key>WallSpace</key>
					<dict>
						<key>ManagedSpaceID</key>
						<integer>214</integer>
						<key>id64</key>
						<integer>215</integer>
						<key>type</key>
						<integer>6</integer>
						<key>uuid</key>
						<string>1058A8F1-0ED1-4295-ABE5-C297222892A6</string>
					</dict>
					<key>fs_wid</key>
					<integer>3534</integer>
					<key>id64</key>
					<integer>214</integer>
					<key>pid</key>
					<integer>379</integer>
					<key>type</key>
					<integer>4</integer>
					<key>uuid</key>
					<string>14D0CFFA-D86B-4097-B638-B8EC48CE573B</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>7</integer>
					<key>id64</key>
					<integer>7</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>924F808D-E93A-4767-A597-B65916155DD1</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>10</integer>
					<key>id64</key>
					<integer>10</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>EAF17BDC-F907-4B4A-B2B7-45712D557DCB</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>13</integer>
					<key>id64</key>
					<integer>13</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>36203216-176E-4064-B676-CAEBD2C387A0</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>14</integer>
					<key>id64</key>
					<integer>14</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>CD10B406-1036-440C-9C9B-03E3EFF3FE35</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>9</integer>
					<key>id64</key>
					<integer>9</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>B6469B88-AA02-4D87-A09A-B973E18DAA1D</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>12</integer>
					<key>id64</key>
					<integer>12</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>7AB4F9C8-16F7-428D-9486-4BE9421372FE</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>8</integer>
					<key>id64</key>
					<integer>8</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>1DE0C2ED-6867-4265-885F-190B1C4925F9</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>11</integer>
					<key>id64</key>
					<integer>11</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>339B9F21-49E3-4D52-98BE-592A3D50483F</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>15</integer>
					<key>id64</key>
					<integer>15</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>FFB0B975-9BB6-4DC8-B368-C071A35FA85B</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>16</integer>
					<key>id64</key>
					<integer>16</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>552AB003-F2A0-4BFA-971C-4395459768E2</string>
				</dict>
				<dict>
					<key>ManagedSpaceID</key>
					<integer>17</integer>
					<key>id64</key>
					<integer>17</integer>
					<key>type</key>
					<integer>0</integer>
					<key>uuid</key>
					<string>DA0A10BA-6D08-4855-9F22-15D9E390A90B</string>
				</dict>
			</array>
		</dict>
	</array>
</dict>
</plist>

**Comment by oakif** [comment]
@dado3212 Doesn't work for me either. Need my preference files as well?

**Comment by oakif** [comment]
I'm using the latest version (1.9.1) by the way!

**Comment by oakif** [comment]
Good news everyone! I was able to fix the issue, at least for myself. Turns out I had the app installed but not the plugin. I simply downloaded the plugin from the GitHub released page, installed it, and it started working perfectly!

**Comment by cweiss2** [comment]
> Good news everyone! I was able to fix the issue, at least for myself. Turns out I had the app installed but not the plugin. I simply downloaded the plugin from the GitHub releases page, installed it, and it started working perfectly!

Can you please link to this? What plugin?

**Comment by oakif** [comment]
Sure, you will find it [here](https://github.com/dado3212/spaces-renamer/releases) once you unzip the downloaded file. Just open the plugin file with MacForce in order to install it to the correct directory (or do it manually as described in MacForce). It should start working immediately.

**Comment by cweiss2** [comment]
It is still not working for me.

**Comment by smashah** [comment]
I got it working by going into finder, deleting anything to do with spaces renamed then installing it from macforge

**Comment by GustafPetersen** [comment]
I have installed the software following the instruction meticulously, and everything is installed. However, my spaces are not renamed.

I have Big Sur version 11.0.1
and running on a MacBook Pro M1, 2020.

Has anyone been able to get it to run on this setup?

**Comment by oakif** [comment]
Maybe the M1 is the issue! I am on an intel Mac and it works fine for me. @cweiss2 are you also on the M-series chip?

**Comment by fabrizziosoares** [comment]
Does anyone have any success on M1 ?
Cheers!

**Comment by client-side96** [comment]
> Good news everyone! I was able to fix the issue, at least for myself. Turns out I had the app installed but not the plugin. I simply downloaded the plugin from the GitHub releases page, installed it, and it started working perfectly!

Worked for me.

**Comment by noel-jean-paul** [comment]
> Good news everyone! I was able to fix the issue, at least for myself. Turns out I had the app installed but not the plugin. I simply downloaded the plugin from the GitHub releases page, installed it, and it started working perfectly!

The 'plugin' being referred to is the spaces-renamer.bundle file in the spaces-renamer folder you download.
Double clicking it will run the file with MacForge and it should appear in your MacForge 'Manage Plugins' tab (see screenshot).

If you do not see spaces-renamer in your MacForge plugins then updating names in the spaces-renamer application in your top drawer will not work.
![Screen Shot 2021-04-13 at 15 47 28](https://user-images.githubusercontent.com/49735398/114494208-135c0500-9c70-11eb-8236-76f941a33f3b.png)

**Comment by dado3212** [comment]
Marking this as closed. This is either resolvable, or handled by #75, with current lack of M1 support.

**Comment by bomtailey** [comment]
I just realized that what's not working for me is seeing the spaces-renamer alternate display of open spaces. I believe I've installed the components correctly (SpacesRenamer is running and the plugin appears in MacForge) but if I show open spaces (e.g. using ^ꜛ) I still see the standard Mac open spaces display (the gray bar at the top listing Desktop 1 ... etc). What am I missing in my installation? Why don't I get the alternate open spaces prompt?
Thanks.

**Comment by dado3212** [comment]
You change space names with the SpacesRenamer app. There should be an icon in the doc for renaming, it's not in the open spaces screen. If you're still having problems, please open a new issue to avoid cluttering this one.

**Comment by kathryntsai** [comment]
Hi!  This seems similar to my issue but I can open another one if this is different -- I'm running Big Sur 11.6 on a Macbook with 2.3 GHz 8-Core Intel Core i9 processor.

I've followed the tutorial -- downloaded MacForge, disabled SIP, downloaded the .zip file, opened the bundle/plugin in MacForge where it appears, double-clicked the Application to move it to the /Applications folder (it shows up in Login items).  The menu bar icon appears and I've been able to type new names and click "Update Names," but the Desktop names still rename the same.  

I tried deleting everything (the bundle from MacForge, the application) and restarting everything, but (a) that doesn't work, and (b) the names I had used in my previous attempt are still there.
